packages/components/src/__actions__/Menu/v3/Menu.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
```

Comment:
Have you tried not using the SCSS syntax for this? I'm pretty sure it would work...

Reply:
Done